exercises/two-bucket/two-bucket.spec.js

```diff
@@ -1,44 +1,44 @@
 var TwoBucket = require('./two-bucket');
 
 describe('TwoBucket', function () {
-  describe('works for input of 3,5,1', function () {
-    var buckOne = 3;
-    var buckTwo = 5;
+  describe('Measure using bucket one of size 3 and bucket two of size 5 - ', function () {
```

Comment:
I didn't check all spec descriptions, but it seems @rootulp takes into account information from the problem-specifications repo. Thanks!